runtime/doc/nvim_terminal_emulator.txt

```diff
@@ -307,6 +307,23 @@ Other commands ~
 	     isn't one
 
 
+Prompt mode ~
+						*termdebug-prompt*
+When on MS-Windows, gdb will run in a buffer with 'buftype' set to "prompt".
+This works slightly differently:
+- The gdb window will be in Insert mode while typing commands.  Go to Normal
```

Comment:
why does it need to be different on Windows?

Reply:
Communicating with `gdb` requires a `pty`, but Windows can not get a `pty`. Therefore, Windows needs prompt buffer.

Reply:
I thought that winpty could be used for that.

Reply:
> I thought that winpty could be used for that.

If `winpty` provides a named pipe connected to a hidden console to pass to `gdb`, there is possibility to work. However, the current `winpty` does not provide such a pipe. Therefore, I think that does not work with the current `winpty`.